### PR TITLE
Create package.gitignore

### PR DESCRIPTION
**Pull request by mosesag0813**

### CODE REVIEW

**Comment by shiftkey** [comment]
@mosesag0813 could you tell me a bit about where these rules are defined?

**Comment by mosesag0813** [comment]
@shiftkey Yea, heres a link as to the repo i am working on. Here

**Comment by mosesag0813** [comment]
Also the rules are defined in the repo

**Comment by shiftkey** [comment]
@mosesag0813 that link seems to 404 - is that right?

**Comment by mosesag0813** [comment]
Weird... Let me try again.

**Comment by mosesag0813** [comment]
Right...the gitignore has changed, the reason for the 404. I may have not done this right, but here the gitignore. Here

**Comment by mosesag0813** [comment]
The repo will be integrated with a custom build.

**Comment by shiftkey** [comment]
I'm going to close this out because I still have no details about the motivation behind this template.

@mosesag0813 if you feel like this should still be a template listed here, please read the section about [specialized templates](https://github.com/github/gitignore#specialized-templates) and open a new pull request with more details.
